### PR TITLE
Removed reliance on site-ai-chat as a bridge to chat-agent

### PR DESCRIPTION
**Pull request by zdodson21**
As title says. As long as there are no issues (I did not encounter any in the test environment or demo, although it might be good if we hook up the test environment to communicate with the backend), the site-ai-chat file can probably be deleted, as it was previously just acting as a bridge between the site builder / editor ui and chat-agent. This pull request makes it a direct connection rather than a "bridged connection".

### CODE REVIEW

File: elements/haxcms-elements/lib/core/haxcms-site-builder.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ class HAXCMSSiteBuilder extends I18NMixin(LitElement) {
   // render function
   render() {
     return html`
-      <site-ai-chat></site-ai-chat>
+      <!-- <site-ai-chat></site-ai-chat> -->
```
Comment (btopro) [inline elements/haxcms-elements/lib/core/haxcms-site-builder.js:84]: just remove not comment

File: elements/haxcms-elements/lib/core/haxcms-site-editor-ui.js [hunk 2]
```diff
@@ -2456,7 +2456,8 @@ class HAXCMSSiteEditorUI extends HAXCMSThemeParts(
     import("@haxtheweb/video-player/lib/lecture-player.js");
     // AI chat bot
     import(
-      "@haxtheweb/haxcms-elements/lib/ui-components/magic/site-ai-chat.js"
```
Comment (btopro) [inline elements/haxcms-elements/lib/core/haxcms-site-editor-ui.js:2459]: what does site-ai-chat do or can it be dropped? agent should work withoutt haxcms; site ai chat was to bridge the 2

**Review by btopro — CHANGES_REQUESTED** [review]

**Comment by zdodson21** [comment]
Just committed requested changes. Deleted commented code and deleted site-ai-chat since it does nothing now.

File: elements/haxcms-elements/lib/core/haxcms-site-editor-ui.js (hunk 2 above)
Comment (zdodson21) [reply elements/haxcms-elements/lib/core/haxcms-site-editor-ui.js:2459]: It was the original AI Chat interface, but everything that was in site-ai-chat was remade into chat-agent and it's subcomponents, so there's no need for it now. Deleted it in my recent commit.

**Comment by btopro** [comment]
@zdodson21 Yes, but I'm saying that site-ai-chat can be tthe bridge so that code like https://github.com/haxtheweb/webcomponents/blob/master/elements/chat-agent/src/chat-agent.js#L18-L19 doesn't live in chat-ai

This would require some minor refactoring but envision a world where chat-agent works in a system that is NOT haxcms  based. site-chat-ai scope then:

- generate an instance of `chat-ai`
- feed it reactions to mobx data changes in the site store like darkMode, userName
- rename the haxcms microservice call that's injected to be some kind of /ai/services sorta thing

In this world; other things theoretically could use the service and HAXcms happens to be consuming it. This is why we do the design patterns as far as appending tags to the body and them talking via events at times. In a world where we'd ever want to use the chat-agent tag and hook it to another AI engine / front-end entirely (think dropping it  into a non-haxcms based course for example).

**Pull request closed without merging**